deploy/Chart/templates/dashboard/service.yaml

```
@@ -0,0 +1,17 @@
+{{- if .Values.dashboard.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: dashboard
+  namespace: "{{ .Release.Namespace }}"
+  labels:
+    app.kubernetes.io/name: dashboard
+    app.kubernetes.io/part-of: radius
+spec:
+  ports:
+    - name: http
+      port: 80
+      targetPort: http
```

This must be port number.

```suggestion
      targetPort: 7007
```

I recommend to define variables for port and targetPort in values.yaml and use them instead of hardcoding port